WebInstall/explainer_cross_domain.md

```diff
@@ -235,27 +239,26 @@ switch (state) {
 ####  **Install Sources manifest field**
 * A new field called `install_sources` will be added to the manifest file to have a control list of sites that can install the app. In its most restrictive case, the developer can specify to not allow installation from any other origin, in which case the PWA conforms to its usual behaviour of only being able to be installed from its same origin.
 
-    * `inquire` field: If supported by the UA, the `inquire` field in the install sources hints at the UA that it can inform that installation origin if the app is installed. **If the browser has an active 'Do Not Track (DNT)', equivalent 'Global Privacy Control (GPC)' setting, or is in Private browsing mode, or is an opinionated browser towards privacy, the `inquire` field is ignored and installation origins will not be allowed to know if that application is installed.**
+    * **`getInstalledApps()` method: If supported by the UA, the `getInstalledApps` method returns a list of the content that has been installed from that installation origin.** This means the installation origin will be able to know which applications it has installed, until cache is cleared. The method returns a list of manifest ids of content installed from the calling origin. Additionally, if the browser has an active 'Do Not Track (DNT)', equivalent 'Global Privacy Control (GPC)' setting, is in Private browsing mode, or is an opinionated browser towards privacy, this is ignored and installation origins will not be allowed to know if that application is installed.
```

Comment:
Consider leaning a bit more into the fact that it's _only_ the apps installed by this origin so that the limitations are *really* clear. 

"This means the installation origin will be able to know which applications it has installed, until cache is cleared. **The installation origin will not be informed of any apps installed by other means, whether via another installation origin, directly through the browser, or by a native app store.**"

Reply:
Added the text specifying that it can ONLY get apps installed from that origin. This makes it clearer and avoids possible misinterpretations around what the method is returning.